pkgs/applications/office/libreoffice/libreoffice-srcs-additions.json

```diff
@@ -1,5 +1,15 @@
 {
   "LIBGLTF": {"subdir": "libgltf/"},
   "ODFVALIDATOR": {"subdir": "../extern/"},
-  "OFFICEOTRON": {"subdir": "../extern/"}
+  "OFFICEOTRON": {"subdir": "../extern/"},
+  "FONT_NOTO_MORE": {"baseurl": "https://noto-website-2.storage.googleapis.com/pkgs"},
```

Comment:
Hmm, this downloads 1.1G of fonts which are then not used because we build with `--without-fonts`. Perhaps implement `skip` instead of `baseurl` to save space & time?